apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -1,23 +1,26 @@
 const dayjs = require('dayjs')
-const { get, uniq, isNull } = require('lodash')
+const locale_ru = require('dayjs/locale/ru')
```

Comment:
Why it's only ru_locale used here?

Reply:
the default locale is English, so to support both, you only need to load ru